analyses/interaction-plots/01-create-interaction-plots.sh

```diff
@@ -40,12 +40,12 @@ combined_plot=${plot_dir}/combined_top50
 declare -A disease
 disease[LGAT]="Low-grade astrocytic tumor"
 if [ "$ALL" -gt "0" ]; then
-  disease[Medulloblastoma]="Medulloblastoma"
-  disease[Ependymoma]="Ependymoma"
-  disease[HGAT]="High-grade glioma"
-  disease[DMG]="Diffuse midline glioma"
-  disease[Ganglioglioma]="Ganglioglioma"
-  disease[Craniopharyngioma]="Craniopharyngioma"
+  disease[Diffuse-astrocytic-and-oligodendroglial-tumor]="Diffuse astrocytic and oligodendroglial tumor"
+  disease[Tumors-of-sellar-region]="Tumors of sellar region"
+  disease[Embryonal-tumor]="Embryonal tumor"
+  disease[Ependymal-tumor]="Ependymal tumor"
+  disease[Meningioma]="Meningioma"
+  disease[Neuronal-mixed-neuronal-glial-tumor]="Neuronal and mixed neuronal-glial tumor"
```

Comment:
In the most recent commit, I swapped `harmonized_diagnosis` with `broad_histology` and updated the shell script as seen above to use the broad histology values. I am not sure if this was the best way to represent these values in the array (using a `-` to separate words), so feedback is welcome here! Also, is this what we want?

Reply:
I would have guessed that you couldn't use `-` in array names, but I would have been wrong! It seems to work just fine.